src/components/ProblemsPage/Status.tsx

```diff
@@ -4,10 +4,12 @@ import UserDataContext from '../../context/UserDataContext/UserDataContext';
 import { useDarkMode } from '../../context/DarkModeContext';
 
 const options = [
+  { label: 'Not Attempted', value: 'Not Attempted' },
```

Comment:
if the labels and values are all the same maybe import from here https://github.com/cpinitiative/usaco-guide/blob/master/src/models/problem.ts#L806?

Reply:
Good idea but i think react-select has some weird requirements where all items need to have a label (for option display) and a value

Reply:
idk how to combat that